R/editEMLfunctions.R

```diff
@@ -2214,18 +2214,26 @@ set_data_urls <- function(eml_object, url = NULL, force = FALSE, NPS = TRUE){
     data_url <- paste0("https://irma.nps.gov/DataStore/Reference/Profile/",
                      ds_ref)
 
-    #update dataset online url. Should remove "function = "download" from ezEML.
-    eml_object$dataset$distribution$online$url <- data_url
+
+    # update dataset online url. Should add `"function = "information"`
+    # and replace inaccurate `function = "download"` from from ezEML.
+    url <- list(url = data_url, `function` <- "information")
```

Comment:
Change `<-` to `=`: ``url <- list(url = data_url, `function` = "information")``
Should fix the "local assignments to syntactic functions" warning